cmd/createdkg.go

```diff
@@ -133,6 +125,9 @@ func runCreateDKG(ctx context.Context, conf createDKGConfig) (err error) {
 
 	forkVersion := networkToForkVersion[conf.Network]
 
+	// Set cluster threshold.
+	conf.Threshold = int(math.Ceil(float64(2*len(conf.OperatorENRs)+1) / float64(3)))
```

Comment:
lets exact this as a function to `cluster` package: `cluster.Threshold()`. Also please use the correct formula (see QBFT package)

Reply:
Only default to this is threshold not set.

Reply:
i have changed it to: set to this if threshold is less than this value. Since users can also set value less than this.

Reply:
best not update values, either use a default or use the value the user provided or error if it is wrong